packages/ember-runtime/lib/mixins/-proxy.js

```diff
@@ -141,6 +135,8 @@ export default Mixin.create({
       !this.isController,
       { id: 'ember-runtime.controller-proxy', until: '3.0.0' }
     );
-    return set(content, key, value);
+    let ret = set(content, key, value);
+    tagFor(this).contentDidChange();
```

Comment:
should this just be `m._tag.contentDidChange` since it is looked up at the start of `setUnknownProperty`?